CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# OntoRefine CLI
+
+## Version 1.1
+
+### New
+
+ - TBD
+
+### Changes
+
+ - Move the project back Java 11. Initially it was on 11, but before the first release it was downgraded to 8 due to development of an internal project. Now we are
+   able to proceed with the initial plan and continue the development process with the current LTS version that is used internally in the organization.
+ - All third party and utility libraries are now updated to the latest version. This is done in order to prevent any security issues due to libraries
+   vulnerabilities. This will be done on regular bases in the future.
+ - The version of the `ontorefine-client` library was updated to the latest in order to use the new functionalities that are developed there and to remove some
+   bugs from the version `1.1.0` that was previously used.
+ - Updated the CI scripts to the latest versions of the GitHub actions. Also we moved from `adopt` to `temurin` Java, because the internally in the organization
+   we are moving to `temurin`.
```

Comment:
Could mention the reason - `adopt` actually moved to `temurin` and it would no longer be supported. https://blog.adoptopenjdk.net/2021/03/transition-to-eclipse-an-update/

Reply:
Fixed.